docs/command-reference/uds_ui.md

```diff
@@ -0,0 +1,37 @@
+---
+title: uds ui
+description: UDS CLI command reference for <code>uds ui</code>.
+type: docs
+---
+## uds ui
+
+[beta] Deploy UDS Runtime and view UI
```

Comment:
It's my understanding we are not "deploying" Runtime here, so if nothing else we should update the wordage. 

```suggestion
[beta] Launch UDS Runtime and view UI
```

Reply:
good catch, these docs are auto generated based on the code, but i'll update the working in lang.go